apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -198,6 +210,28 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
             />
           </div>
         )}
+        {!useEventTypeDestinationCalendarEmail && verifiedSecondaryEmails.length > 0 && (
+          <div className="w-full">
+            <SelectField
+              label={t("send_event_details_to")}
+              placeholder={
+                selectedSecondaryEmailId === -1 && (
+                  <span className="text-default min-w-0 overflow-hidden truncate whitespace-nowrap">
+                    <Badge variant="blue">Default</Badge> {user?.email || ""}
+                  </span>
+                )
+              }
+              onChange={(option) =>
+                formMethods.setValue("secondaryEmailId", option?.value, { shouldDirty: true })
+              }
+              value={verifiedSecondaryEmails.find(
+                (secondaryEmail) =>
+                  selectedSecondaryEmailId !== -1 && secondaryEmail.value === selectedSecondaryEmailId
+              )}
+              options={verifiedSecondaryEmails}
+            />
+          </div>
+        )}
```

Comment:
We should ensure this works with @Udit-takkar PR with the option of choosing destination calendar emails.

Reply:
@joeauyeung I have tested this from my end and looks fine, @Udit-takkar Could you please test this PR to make sure your changes are not affected. Thanks